examples/api/lib/material/switch_list_tile/custom_labeled_switch.0.dart

```diff
@@ -0,0 +1,100 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for custom labeled switch.
+
+import 'package:flutter/gestures.dart';
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const LabeledSwitchApp());
+
+class LabeledSwitchApp extends StatelessWidget {
+  const LabeledSwitchApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(useMaterial3: true),
+      home: Scaffold(
+        appBar: AppBar(title: const Text('Custom Labeled Switch Sample')),
+        body: const Center(
+          child: LabeledSwitchExample(),
+        ),
+      ),
+    );
+  }
+}
+
+class LinkedLabelSwitch extends StatelessWidget {
+  const LinkedLabelSwitch({
+    super.key,
+    required this.label,
+    required this.padding,
+    required this.value,
+    required this.onChanged,
+  });
+
+  final String label;
+  final EdgeInsets padding;
+  final bool value;
+  final ValueChanged<bool> onChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: padding,
+      child: Row(
+        children: <Widget>[
+          Expanded(
+            child: RichText(
+              text: TextSpan(
+                text: label,
+                style: const TextStyle(
+                  color: Colors.blueAccent,
```

Comment:
It would be nice to use a color scheme color for cases like this. That way if the user tries dark mode, this color will update.